bundles/action/org.openhab.action.pushover/README.md

```diff
@@ -40,13 +40,39 @@ For specific information on each item, see the [Pushover API](https://pushover.n
 - `pushover(String apiToken, String userKey, String message, String device, int priority)`
 - `pushover(String apiToken, String userKey, String message, String device, String title, String url, String urlTitle, int priority, String soundFile)`
 
+- `sendPushoverEmergency(String message)`

```

Comment:
Question: Should we add the type of the return value in front of the actions?

Reply:
I suspect you didn't do so because you didn't want to break the scheme. How about:

- `sendPushoverEmergency(String message)` (returns String, please see below)


---

Or maybe better:

>The `sendPushoverEmergency` actions will send an message with [Emergency Priority](https://pushover.net/api#priority). The actions return a `receipt` identifier you may store in a String variable. The identifier is used with the `cancelPushoverEmergency` actions to cancel the notification prior to reaching the `defaultExpire` value  of 3 hours.
>
>- `sendPushoverEmergency(String message, String title)`
>- `sendPushoverEmergency(String message, String title, String device)`
>- `sendPushoverEmergency(String message, String title, String device, String soundFile)`
>
><a>
>    
>- `cancelPushoverEmergency(String receipt)`
>- `cancelPushoverEmergency(String apiToken, String userKey, String receipt)`